src/test/codegen/vec-calloc.rs

```diff
@@ -1,32 +1,86 @@
 // compile-flags: -O
 // only-x86_64
-// ignore-debug
```

Comment:
Removed this because it was suggested for me on Zulip. Works on my machine without it.